polyfill/test/zoneddatetime.mjs

```diff
@@ -1064,6 +1064,30 @@ describe('ZonedDateTime', () => {
     timeZone: tz
   });
   describe('properties around DST', () => {
+    it('hoursInDay works with DST start', () => {
+      equal(hourBeforeDstStart.hoursInDay, 23);
+    });
+    it('hoursInDay works with non-DST days', () => {
+      equal(dayBeforeDstStart.hoursInDay, 24);
+    });
+    it('hoursInDay works with DST end', () => {
+      const dstEnd = ZonedDateTime.from('2020-11-01T01:00-08:00[America/Los_Angeles]');
+      equal(dstEnd.hoursInDay, 25);
+    });
+    it('hoursInDay works with non-hour DST change', () => {
+      const zdt1 = ZonedDateTime.from('2020-10-04T12:00[Australia/Lord_Howe]');
+      equal(zdt1.hoursInDay, 23.5);
+      const zdt2 = ZonedDateTime.from('2020-04-05T12:00[Australia/Lord_Howe]');
+      equal(zdt2.hoursInDay, 24.5);
+    });
+    it('hoursInDay works with non-half-hour DST change', () => {
+      const zdt = ZonedDateTime.from('1933-01-01T12:00[Asia/Singapore]');
+      assert(Math.abs(zdt.hoursInDay - 23.6666666666666666) < Number.EPSILON);
```

Comment:
I had no idea that `Number.EPSILON` existed.  Very useful. Thanks for the education!

Reply:
I'd actually prefer if the test framework provided an `assert.almostEqual()` for floats, as Jasmine does, but this works well enough.